service/config.go

```diff
@@ -55,3 +58,11 @@ type CRIKeychainConfig struct {
 
 // ResolverConfig is config for resolving registries.
 type ResolverConfig resolver.Config
+
+// SnapshotterConfig is snapshotter-related config.
+type SnapshotterConfig struct {
+	// NoRestoreInvalid doesn't restore invalid snapshots.
```

Comment:
What does "restore" mean

Reply:
Fixed the patch not to use "restore" but use clearer expression.